Plugins/BackbonePluginTemplatePlugin/BackbonePluginTemplatePluginGenerator.cs

```diff
@@ -131,6 +140,87 @@ private GlobalJsonModification CreateMsBuildSdk()
             return globalJsonModificationBuilder.Build();
         }
 
+        private Task AddProjectReferences()
+        {
+            var packagesReferences = CreateProjectModification();
```

Comment:
Rename this to `modification`

Reply:
Addressed in: [Rename local variable](https://github.com/messerli-informatik-ag/meta-generator/pull/27/commits/4fb7b0814fac55eff721025c7b6019e7a8479caf)